tests/end2end/features/tabs.feature

```diff
@@ -1535,9 +1535,10 @@ Feature: Tab management
         When I open data/numbers/1.txt
         And I run :tab-pin
         And I open data/numbers/2.txt without waiting
```

Comment:
```suggestion
        And I open data/numbers/2.txt
```

so that the test now actually waits for the new tab to be opened.

Reply:
Okay will do in next commit.